apps/codeforafrica/src/lib/api.ghost/index.js

```diff
@@ -35,15 +35,11 @@ export async function getStory(slug) {
   return getPost(slug);
 }
 
-export async function getRelatedStories(story) {
-  if (!story?.tags?.length) {
-    return null;
-  }
-
+export async function getRelatedStoriesByTags(tags, story = {}) {
```

Comment:
```suggestion
export async function getRelatedStoriesByTags(tags) {
```

Why do we still need `story`?

Reply:
To prevent showing current story under `News and Stories`

Reply:
We can `filter`/`slice` the returned list similar to how we're doing for related projects @esirK .